Assets/_Project/Scripts/Level/Storage/RemoteLevelStorageProviderAws.cs

```diff
@@ -248,14 +286,23 @@ public void LoadAllLevelData(RemoteLevelStorage_AllLevelsLoadedCallback callback
         {
             string limit = "10";
             string cursor = "";
+            string code = "";
+            string sort = "";
+            SortDirection direction = SortDirection.Ascending;
             if (query != null)
             {
                 limit = query.Value.limit.ToString();
                 cursor = query.Value.cursor;
+                code = query.Value.code;
+                sort = query.Value.sort;
+                direction = query.Value.direction;
             }
 
             string queryParams = $"?limit={limit}";
             queryParams += cursor.Length > 0 ? $"&cursor={cursor}" : "";
+            queryParams += code.Length > 0 ? $"&code={code}" : "";
+            queryParams += sort.Length > 0 ? $"&sortKey={sort}" : "";
+            queryParams += direction == SortDirection.Ascending ? "&sortOrder=ASC" : "&sortOrder=DESC";
 
```

Comment:
TODO: I need to double check what the correct query params are in the API to make this work.